apps/src/music/views/Callouts.tsx

```diff
@@ -72,55 +79,93 @@ const Callouts: React.FunctionComponent = () => {
     (state: {music: MusicState}) => state.music.currentCallout
   );
 
-  const availableCallout = callout.id
-    ? availableCallouts[callout.id]
-    : undefined;
+  const [isMoving, setIsMoving] = useState(false);
+
+  const calloutIds = callout?.id?.split('---');
+
+  const validCallouts: AvailableCallout[] | undefined = calloutIds?.map(
+    calloutId => availableCallouts[calloutId]
+  );
 
   const calloutIndex = callout.index;
 
+  interface Target {
+    left: number;
+    top: number;
+  }
+
+  const targets: Target[] = [];
+
+  validCallouts?.map(validCallout => {
+    const element = document.querySelector(validCallout.selector);
+    const elementRect = element?.getBoundingClientRect();
+    if (elementRect && elementRect.width > 0) {
+      const elementWidth = elementRect.right - elementRect.left + 1;
+      const target: Target = {
+        left: elementRect.left + elementWidth / 2,
+        top: elementRect.bottom + 4,
+      };
+      targets.push(target);
+    }
+  });
+
+  const openToolboxCategory =
+    validCallouts && validCallouts[0].openToolboxCategory;
```

Comment:
Is there a chance the second target would need to open a toolbox category?

Reply:
I can't think of a case when we'd show a drag _into_ a category, so I think we're okay keeping it simple like this for now.